spidermon/contrib/scrapy/monitors.py

```diff
@@ -374,6 +380,53 @@ def test_check_if_field_coverage_rules_are_met(self):
         self.assertTrue(len(failures) == 0, msg=msg)
 
 
+@monitors.name("Jobs Comparison Monitor")
+class JobsComparisonMonitor(BaseScrapyMonitor):
+    """Check for a drop in scraped item count compared to previous jobs.
+
+    You need to set the number of previous jobs to compare, using ``SPIDERMON_JOBS_COMPARISON``.
+    The default is ``0`` which disables the monitor. We use the average of the scraped items count.
+
+    You can configure which percentage of the previous item count is the minimum acceptable, by
+    using the setting ``SPIDERMON_JOBS_COMPARISON_THRESHOLD``. We expect a float number between
+    ``0.0`` and ``1.0``. The default is ``0.8``.
+
+    You can filter which jobs to compare based on their states using the
+    ``SPIDERMON_JOBS_COMPARISON_STATES`` setting. The default value is ``("finished",)``.
+    """
+
+    @property
+    def client(self):
```

Comment:
**question** Where this property is being used?

Reply:
[A few lines below](https://github.com/scrapinghub/spidermon/pull/323/files#diff-9d830f5588694dea71ec34a158727c22a20d77630f7d8afd1705d966241ab00dR413).

It is just a wrapper for the hs client (which I'm still amending). I made it a property for an easier mocking.